ResetPasswordBundle/src/tests/UnitTests/PasswordResetHelperTest.php

```diff
@@ -0,0 +1,330 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace SymfonyCasts\Bundle\ResetPassword\tests\UnitTests;
+
+use SymfonyCasts\Bundle\ResetPassword\Exception\ExpiredResetPasswordTokenException;
+use SymfonyCasts\Bundle\ResetPassword\Exception\InvalidResetPasswordTokenException;
+use SymfonyCasts\Bundle\ResetPassword\Exception\TooManyPasswordRequestsException;
+use SymfonyCasts\Bundle\ResetPassword\Generator\TokenGenerator;
+use SymfonyCasts\Bundle\ResetPassword\Model\PasswordResetRequestInterface;
+use SymfonyCasts\Bundle\ResetPassword\PasswordResetHelper;
+use SymfonyCasts\Bundle\ResetPassword\Persistence\PasswordResetRequestRepositoryInterface;
+use SymfonyCasts\Bundle\ResetPassword\tests\Fixtures\PasswordResetRequestTestFixture;
+use SymfonyCasts\Bundle\ResetPassword\tests\Fixtures\UserTestFixture;
+use SymfonyCasts\Bundle\ResetPassword\tests\UnitTests\Model\AbstractModelUnitTest;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ */
+class PasswordResetHelperTest extends AbstractModelUnitTest
+{
+    protected $sut = PasswordResetHelper::class;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|PasswordResetRequestRepositoryInterface
+     */
+    protected $mockRepo;
+
+    /**
+     * @var string
+     */
+    protected $tokenSigningKey;
+
+    /**
+     * @var int
+     */
+    protected $resetRequestLifetime;
+
+    /**
+     * @var int
+     */
+    protected $requestThrottleTime;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|TokenGenerator
+     */
+    protected $mockGenerator;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|PasswordResetRequestInterface
+     */
+    protected $mockResetRequest;
+
+    /**
+     * @var string
+     */
+    protected $randomToken;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|UserTestFixture
+     */
+    protected $mockUserFixture;
```

Comment:
All private

And if, for some reason, we *don't* ever need one of these values in a test method - like I think `tokenSigningKey` - then don't add it as a property.

Reply:
done